Sources/Jobs/JobMetricsHelper.swift

```diff
@@ -46,21 +50,9 @@ internal enum JobMetricsHelper {
         error: Error? = nil,
         retrying: Bool = false
     ) {
-        // This meter can be used to display total job
-        // Or decrement processing vector in Prometheus UI or Grafana
-        // with something like count(swif_jobs_meter{status="processing"}
-        // unless on(jobID) (swif_jobs_meter{status="queued"})
-        // or (swif_jobs_meter{status="completed")) or vector(0)
-        Meter(
-            label: JobMetricsHelper.meterLabel,
-            dimensions: [
-                ("status", JobMetricsHelper.JobStatus.completed.rawValue)
-            ]
-        ).increment()
```

Comment:
This value in this meter can be accessed from counter `swift.jobs` with dimension status set to completed